presto-main/src/main/java/com/facebook/presto/server/TaskResource.java

```diff
@@ -267,6 +306,12 @@ public TaskInfo deleteTask(
         return taskInfo;
     }
 
+    /**
+     * shuffle related
+     *
+     * The downstream task gets the output data from a buffer from the specific upstream task.
```

Comment:
perhaps, "Returns the results produced by the specified task. The results are partitioned and bufferId parameter specifies the partition for which to return the data."

It would be helpful to describe the bufferId, token and maxSize parameters.